src/app/page.js

```diff
@@ -94,20 +92,56 @@ export default function Home() {
         </h1>
         <div className="grid lg:grid-cols-2 lg:grid-rows-1 sm:grid-cols-1 sm:grid-rows-2 gap-4 lg:gap-8 sm:gap-4 my-20 text-gray-400 font-mono">
           <p>
-            In the dynamic landscape of blockchain technology, where the
-            efficiency and reliability of RPC nodes are paramount; traditional
-            tools often need to catch up by focusing mostly on latency. At
-            Chainstack, we recognize the necessity of a more holistic approach
-            to node performance evaluation.
+            Chainstack Compare is tool to measure the node performance metrics
+            that make sense.<br/>
+            The tool runs server-side on an instance maintained by Chainstack.<br/>
+            The tool does a few calls to the exact same dummy contracts deployed
+            on a few EVM networks.
+            The tool measures the performance from three data points:<br/>
+            i) the number of processed blocks with the calls;<br/>
+            ii) the time it takes to process the blocks with the calls;<br/>
+            iii) the resulting blocks-per-second metric, or the expected
+            data throughput.<br/>
+            As you can see, all the parameters are uninform except for the one
+            you care about and feed to the tool — your node endpoint.
           </p>
           <p>
-            Chainstack Compare transcends conventional metrics by taking a
-            different approach to measuring node efficiency, particularly in
-            data fetching — a critical factor for the performance of
-            decentralized applications. Designed to the modern needs of
-            developers, our tool provides a nuanced assessment that goes beyond
-            simple latency numbers to capture the essence of real-world node
-            performance.
+            Server-side instance specs:<br/>
+            Contract: <a href="https://etherscan.io/address/0x087CDFb5D353395Cff09d986295d59A0B9E15D77#code" 
+            className="text-blue-500 hover:text-blue-700 underline" 
+            target="_blank" 
+            rel="noopener noreferrer">
+            CompareNodePerformance
+            </a><br/>
```

Comment:
@kolebayev @soos3d this is basically the info that needs to be there. if you can make it nicer, I'd appreciate it. mine is looking horrible obviously